mappings/net/minecraft/entity/attribute/AttributeContainer.mapping

```diff
@@ -24,3 +24,11 @@ CLASS net/minecraft/class_5131 net/minecraft/entity/attribute/AttributeContainer
 	METHOD method_26855 toTag ()Lnet/minecraft/class_2499;
 	METHOD method_26856 getBaseValue (Lnet/minecraft/class_1320;)D
 		ARG 1 attribute
+	METHOD method_27305 hasModifierAttribute (Lnet/minecraft/class_1320;Ljava/util/UUID;)Z
```

Comment:
`hasModifierForAttribute`? otherwise sounds like there is a "modifier attribute"